Gui/opensim/build.xml

```diff
@@ -331,9 +331,9 @@
                 includes="**/jxbrowser-*mac*.jar"/>
             <fileset dir="${opensim.dist.dir}/OpenSim/modules/ext"
                 includes="**/jxbrowser-*win*.jar"/>
-            <fileset file="${opensim.dist.dir}/bin/OpenSim.exe"/>
+            <fileset file="${opensim.dist.dir}/bin/opensim.exe"/>
```

Comment:
Not sure that the .exe name case changed.

Reply:
@halleysfifthinc The name actually changed on windows to opensim.exe so I don't think it's failing to delete the file. I'll apply your inlining suggestion. Thank you 👍